specs/engine.spec/src/main/scripts/io/aklivity/zilla/specs/engine/schema/engine.schema.json

```diff
@@ -437,35 +407,6 @@
                 "type",
                 "kind"
             ],
-            "anyOf":
-            [
-                {
-                    "properties":
-                    {
-                        "kind":
-                        {
-                            "const": "remote_server"
-                        }
-                    },
-                    "required":
-                    [
-                        "entry"
-                    ]
-                },
-                {
-                    "properties":
-                    {
-                        "kind":
-                        {
-                            "not":
-                            {
-                                "const": "remote_server"
-                            }
-                        },
-                        "entry": false
-                    }
-                }
-            ],
```

Comment:
Why did we remove this one?
I think this is trying to enforce generic intent of `remote_server` at binding level to make sure any bindings using `remote_server` do so in a consistent manner instead of treating them as independent.

Reply:
I was adding `"catalog": false` to most of the bindings so I also added `"entry": false` moving the logic for this into the individual schemas. This specific section is not needed.

I can revert this and implement the catalog logic differently if you want. I went with this since it got rid of some of the complexity of include or exclude global properties.

Reply:
I think the issue is that we want a guarantee that it's not possible to diverge accidentally regarding `entry`.
The `entry` property only has meaning when `kind` is `remote_server`, so if possible, I'd like to keep that logic defined once by engine to ensure consistency.